idd/Energy+.idd.in

```diff
@@ -52147,6 +52147,11 @@ UnitarySystemPerformance:Multispeed,
        \maximum 10
        \note Used only for Multi speed coils
        \note Enter the number of the following sets of data for air flow rates.
+  A2 , \field Single Mode Operation
+       \type choice
```

Comment:
Thanks for good additions. Here is my wording. Hope it makes sense.

       \note Controls coil operation during each HVAC timestep. 
       \note This choice does not apply to spped 1 opeation.
       \note Yes = operate at the higher speed possible without exceeding the current load.
       \note No = allow operation at the average of two adjacent speeds to match the current load.